src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisAjaxController.java

```diff
@@ -263,12 +265,15 @@ public AnalysisInputFiles ajaxGetAnalysisInputFiles(@PathVariable Long submissio
 
 		for (SampleSequencingObject sso : sampleFiles) {
 			SequenceFilePair fp = (SequenceFilePair) sso.getSequencingObject();
-			if (fp.getFiles().size() == 2) {
+			if (fp.getFiles()
```

Comment:
With the updated formatter that got merged into dev a while back it would not split this line. Could you reformat this file, thanks.

Reply:
I ran the formatter and it didn't change it